api/layer/multipart_upload.go

```diff
@@ -228,10 +228,6 @@ func (n *layer) uploadPart(ctx context.Context, multipartInfo *data.MultipartInf
 		HashSum: partInfo.ETag,
 	}
 
-	if err = n.objCache.PutObject(objInfo); err != nil {
-		n.log.Error("couldn't cache system object", zap.Error(err))
-	}
```

Comment:
Useless because it is a part of multipart object?

Reply:
Useless because all required part info is stored in tree service